src/operator/quantization/mkldnn/mkldnn_quantized_conv.cc

```diff
@@ -0,0 +1,229 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file mkldnn_quantized_onvolution.cc
+ * \brief
+ * \author Wenting Jiang
+*/
+
+#if MXNET_USE_MKLDNN == 1
+#include "../../nn/mkldnn/mkldnn_base-inl.h"
+#include "../../nn/convolution-inl.h"
+#include "../quantization_utils.h"
+#include "../../tensor/matrix_op-inl.h"
+#include "../../elemwise_op_common.h"
+namespace mxnet {
+namespace op {
+
+static mkldnn::convolution_forward::primitive_desc GetConvFwdImpl(
```

Comment:
This is mkldnn API which doesn't differentiate quantized convolution from the name, and we do leveraged many FP32 convolution code(the difference is scale and min/max calculation) and we are considering to share the code in the future PR. For this PR, we prefer to separate them just like what GPU version quantization did.

Reply:
i'm not sure if we are talking about the same thing. My question is why we need to have another `GetConvFwdImpl` here? It seems to me that this function is the same as the one for normal convolution. If they are different, can you point out where the difference is?

It's hard to maintain an extra copy of a function. Functions will be changed. If the difference is very small, I think we should reuse the original function.

It also helps review. Reviewers don't need to read many lines of code so carefully to figure out the difference.

Reply:
I totally agree, the mainly difference is for int8 inference is_train will be always false but this could share code with FP32 version. I will update a new version to share the code with FP32 convolution/pooling.